app/src/main/res/layout/activity_profile.xml

```diff
@@ -4,6 +4,15 @@
     android:layout_height="match_parent"
     android:background="@color/colorWhite">
 
+    <Button
+        android:id="@+id/dotMenu"
+        android:background="@drawable/ic_more_vert_black_24dp"
+        android:layout_width="30dp"
+        android:layout_height="30dp"
+        android:textColor="@color/colorWhite"
```

Comment:
Is line 12 needed attribute? Can you explain what this does?

Reply:
Actually it doesn't work at all. I'll remove this.